src/crimson/osd/pg.h

```diff
@@ -477,6 +477,10 @@ class PG : public boost::intrusive_ref_counter<
     void trim(const pg_log_entry_t &entry) override {
       // TODO
     }
+    void partial_write(pg_info_t *info, const pg_log_entry_t &entry) override {
+      // TODO
+      ceph_abort_msg("not implemented yet");
```

Comment:
We seem to call this for every log entry for all pool types upon removal from the log to update the pwlc information in pg_info_t.  The implementation ends up being a noop as long as there aren't any actual partial_writes (entries with ```!entries.written_shards.empty()```) in the log, but this abort is still being tripped in crimson because it does get called.  I think this abort needs to be changed to something like ```ceph_assert(entry.written_shards.empty() && info->partial_writes_last_complete.empty());```.

See https://tracker.ceph.com/issues/70886#note-16

Fixing this shouldn't require rerunning the normal rados suite, just the crimson one.